packages/utils/project.json

```diff
@@ -36,13 +36,8 @@
       "executor": "@nx/vite:test",
       "outputs": ["{workspaceRoot}/coverage/{projectRoot}"],
       "options": {
-        "passWithNoTests": true
-      },
-      "configurations": {
-        "ci": {
-          "ci": true,
-          "codeCoverage": true
-        }
+        "passWithNoTests": true,
+        "reportsDirectory": "../../coverage/packages/utils"
```

Comment:
```suggestion
        "reportsDirectory": "../../dist/coverage/utils"
```

Reply:
Would it be beter to have all in dist?

Reply:
I would keep it in the default Nx location, it's also gitignored. :man_shrugging:

Reply:
I would keep it in the default Nx location, it's also gitignored. 🤷‍♂️

Reply:
Seems to be ignored so it should be okay.